identity/src/commonMain/kotlin/com/android/identity/nfc/HandoverRequestRecord.kt

```diff
@@ -14,9 +16,10 @@ data class HandoverRequestRecord(
 ) {
 
     fun generateNdefRecord(): NdefRecord {
+        // TODO: 393388370 - Redundant check, but could point at the need of a custom error message param in Byte methods?
```

Comment:
nit: also, the TODO style should be a (short) link, not just a number. So b/393388370 instead of just 393388370.